calling-apis/post-requests-with-axios.md

```diff
@@ -0,0 +1,221 @@
+# POST Requests With axios
+
+## Introduction
+
+Matteo is making the front-end for the **Ada Trip Reserving Service™**. This service:
+
+1. helps tour guides create Trip packages catered to global travelers
+2. helps travelers find Trip packages that fit their needs, and make a reservation for that trip
+
+Matteo's front-end should make calls to [the back-end Ada Trip Reserving Service™ Trip API™™™](https://github.com/AdaGold/trip-api). The first feature he needs to build is enabling users to create new trip packages.
+
+Matteo's first step is to [research the API documentation](https://github.com/AdaGold/trip-api). He'll use the documentation and Postman to answer the following questions:
+
+1. What is the API call that lists all existing Trip packages?
+2. How many existing trips are there?
+3. What is the API call that creates a new Trip package?
+4. What is the verb and URL of that request?
+5. What are the required query parameters? What are the optional query parameters?
+
+<br/>
+
+<details style="max-width: 700px; margin: auto;">
+
+  <summary>
+    Here is what Matteo discovers with the documentation and Postman.
+  </summary>
+
+1. `get` `https://trektravel.herokuapp.com/trips`
+1. Depends, but Matteo will note the shape of the expected JSON response
+1. `post` `https://trektravel.herokuapp.com/trips`
+1. `post`, and `https://trektravel.herokuapp.com/trips`
+1. Required: `name`, `continent`, `category`, `weeks`, and `cost`. Optional: `about`
+
+</details>
+
+## Making a `POST` request
+
+The syntax for making a POST request using `axios` is very similar to making a GET request, but we will use the `post` function instead.
```

Comment:
some code formatting
keep the parallels all the way through
```suggestion
The syntax for making a `POST` request using `axios` is very similar to making a `GET` request, but instead of using the `get` function, `axios` provides a `post` function.
```